### DISCUSSION TITLE
[灌水]六分街那个录像店怎么样

### DISCUSSION THREAD

**Discussion by IIID123** [share121/inter-knot#1619] [其他]
![20240812115328](https://github.com/user-attachments/assets/80751cb8-43fc-4688-a238-bd838fe7ce03)
六分街那个录像店怎么样，种类全不全，想去借点片子看

**Comment by IIID123** [comment]
这个委托我接了！

**Comment by ENDer-Z1** [comment]
听说二楼还有那种片子呢，真的，出家人不打诳语

**Comment by ZhuYuan-148** [comment]
两位店长都很热情呢，总会帮我解决诉求，前台的邦布也很可爱！

**Comment by yutong903** [comment]
两位店长很好看=）
